templates/02-executable-org.md

```diff
@@ -56,7 +57,8 @@ The number (#) should represent the number of proposals the Org has submitted to
 ## **Reporting & Success Criteria**
 >👉 What metrics for success and accountability mechanisms are being put in place with this Org? How can Radworks evaluate its progress? 
 
-> ❗️ If there is an MOU in place with Radworks, link it here. 
+> ❗️ If there is an MOU in place with Radworks, link it here.
```

Comment:
We have to change MOU to partnership agreement if we decide to change the name of the document. Not relevant now, but let's keep it in mind.